misc/ubuntu-touch/kaidan.desktop

```diff
@@ -0,0 +1,8 @@
+[Desktop Entry]
+Type=Application
+Version=1.0
+Name=Kaidan
+Exec=bin/kaidan
+Icon=kaidan.svg
+Terminal=false
+X-Ubuntu-Touch=true
```

Comment:
Could you rebase your desktop file on the normal?

Reply:
Or even better: You could also copy the normal desktop file and just add "X-Ubuntu-Touch" to it.

Reply:
I don't think the other isn't displayed anywhere, it could be displayed. I don't think it is uncommon to show a description.

Also there are other properties that setting the windows class name, etc.

Anyway if you only copy the desktop file and add `X-Ubuntu-Touch`, then we will only have one desktop file, which is better, I think.

Reply:
Also the icon and the Exec entry is different. I don't think it's worth it adapting the desktop file using a shell script.

Reply:
Ok, didn't see that.
Then please add the following properties:
```
GenericName=Jabber IM Client
GenericName[be]=Jabber-праграма
GenericName[be@latin]=Klijent kamunikacyi dla Jabber
GenericName[bg]=Джабър клиент за бързи съобщения
GenericName[br]=Flaperez Jabber
GenericName[cs]=Jabber IM klient
GenericName[da]=Jabber IM Klient
GenericName[de]=Jabber IM-Client
GenericName[el]=Ένα πρόγραμμα για το Jabber
GenericName[en_GB]=Jabber IM Client
GenericName[eo]=Jabber-tujmesaĝilo
GenericName[es]=Cliente de IM Jabber
GenericName[eu]=IM Jabber Bezeroa
GenericName[fr]=Client de Messagerie Instantanée Jabber
GenericName[gl]=Cliente de MI de Jabber
GenericName[he]=לקוח מסרים מידיים Jabber
GenericName[hr]=Jabber IM Klijent
GenericName[hu]=Jabber Azonnali üzenetküldő kliens
GenericName[it]=Client IM Jabber
GenericName[ja]=Jabber IM クライアント
GenericName[kk]=Jabber IM Клиенті
GenericName[lt]=Jabber IM Klientas
GenericName[nb]=Jabber IM Klient
GenericName[nl]=Jabber IM Client
GenericName[no]=Jabber IM Klient
GenericName[pl]=Klient komunikatora Jabber
GenericName[pt]=Cliente de IM Jabber
GenericName[pt_BR]=Um cliente de IM Jabber
GenericName[ru]=Jabber-клиент для мгновенных сообщений
GenericName[sk]=Jabber IM klient
GenericName[sr]=Џабер IM Клијент
GenericName[sr@Latn]=Džaber IM Klijent
GenericName[sv]=Jabber-snabbmeddelandeklient
GenericName[uk]=Клієнт служби миттєвих повідомлень Jabber
GenericName[zh_CN]=Jabber 即时消息客户端
GenericName[zh_TW]=Jabber 即時訊息客戶端
Comment=A QtQuick Jabber client
Comment[be]=Jabber-праграма для QtQuick
Comment[be@latin]=Klijent Jabber dla QtQuick
Comment[bg]=Джабър клиент за QtQuick
Comment[br]=Ur flaperez Jabber e QtQuick
Comment[cs]=QtQuick Jabber klient
Comment[da]=En QtQuick Jabber klient
Comment[de]=Ein Jabber-Client in QtQuick
Comment[el]=Ένα πρόγραμμα για το Jabber γραμμένο σε QtQuick
Comment[en_GB]=A QtQuick Jabber client
Comment[eo]=QtQuickbazita jabber-kliento
Comment[es]=Un cliente de Jabber en GTK
Comment[eu]=QtQuick Jabber bezeroa
Comment[fr]=Un client Jabber en QtQuick
Comment[gl]=Un cliente de Jabber de GTK
Comment[he]=לקוח ג׳אבּר QtQuick‎
Comment[hr]=QtQuick Jabber klijent
Comment[hu]=Egy QtQuick Jabber kliens
Comment[it]=Un client Jabber QtQuick
Comment[ja]=QtQuick Jabber クライアント
Comment[kk]=QtQuick Jabber қолданбасы
Comment[lt]=QtQuick Jabber klientas
Comment[nb]=En QtQuick Jabber klient
Comment[nl]=Een QtQuick Jabber client
Comment[no]=En QtQuick Jabber klient
Comment[pl]=Klient Jabbera w QtQuick
Comment[pt]=Cliente Jabber QtQuick
Comment[pt_BR]=Um cliente Jabber QtQuick
Comment[ru]=Jabber-клиент на QtQuick
Comment[sk]=QtQuick Jabber klient
Comment[sr]=QtQuick Џабер клијент
Comment[sr@Latn]=QtQuick Džaber klijent
Comment[sv]=En QtQuick-baserad Jabber-klient
Comment[uk]=Jabber клієнт, що базується на QtQuick
Comment[zh_CN]=一个 QtQuick 的 Jabber 客户端
Comment[zh_TW]=一個 QtQuick 的 Jabber 客戶端
Keywords=chat;messaging;im;jabber;xmpp;qt;network;
Categories=Network;
```